src/client/sandboxed-jquery.js

```diff
@@ -1719,7 +1719,11 @@ export function init (window, undefined) {
                     container.style.zoom = 1;
                 }
 
-                body.removeChild(container);
+                // NOTE: in some cases Firefox can't remove container div,
```

Comment:
I suppose we should add a more specific comment here, like:
`Sometimes in Firefox the "body.insertBefore(container, body.firstChild)" script above doesn't insert a container (it looks it depends on the page loading phase). Since this behavior is unstable we should check is body contains the container`